src/_locales/ru/messages.json

```diff
@@ -29,23 +29,23 @@
     "html_about_github_title": { "message": "GitHub" },
     "html_about_twitter_title": { "message": "Twitter" },
     "html_about_report_issues_prefix": { "message": "Связаться с нами или сообщить о баге можно на" },
-    "html_about_report_issues_suffix": { "message": ". Пожалуйста. убедитесь что ваша идея или предложение не опубликованы ранее. Спасибо!" },
+    "html_about_report_issues_suffix": { "message": ". Пожалуйста, убедитесь, что Ваша идея или предложение не были опубликованы ранее. Спасибо!" },
     "html_about_debugging_prefix": { "message": "Чтобы включить ведение журнала или установить флаги отладки, перейдите на " },
     "html_about_debugging_suffix": { "message": "страницу отладки." },
     "html_about_the_github_issues_page": { "message": "странице предложений GitHub" },
     "html_about_html2canvas": { "message": "html2canvas" },
     "html_about_html2canvas_prefix": { "message": "Это расширение использует" },
     "html_about_html2canvas_suffix": { "message": "библиотеку Niklas von Hertzen." },
     "html_about_indexedDb": { "message": "db.js" },
-    "html_about_indexedDb_prefix": { "message": "Также используется обёртка indexedDb " },
+    "html_about_indexedDb_prefix": { "message": "Также используется indexedDb враппер " },
```

Comment:
This is Anglicism (transcription of "wrapper" word) instead of translation ("обёртка"). I think, the previous one is better.

Reply:
```suggestion
    "html_about_indexedDb_prefix": { "message": "Также используется обёртка indexedDb " },
```

Reply:
We with @MonsterNya agreed on anglicism "враппер" [here](https://github.com/deanoemcke/thegreatsuspender/pull/848#pullrequestreview-199113190) coz of  the specificity of the term and the target audience of this exact line, so let's leave it as it is imo

Reply:
> We with @MonsterNya agreed on anglicism "враппер" [here](https://github.com/deanoemcke/thegreatsuspender/pull/848#pullrequestreview-199113190) coz of the specificity of the term and the target audience of this exact line, so let's leave it as it is imo

"враппер" is slang. In (semi-)official docs "обёртка" is used. I'm not even sure that "враппер" word actual exists: https://ru.wiktionary.org/wiki/враппер

It sounds like "постгря" for PostgreSQL. I'm against these slang words out of speech (in writing, especial public).

Reply:
So if u say it's being used in docs, well let's leave urs actually original variant then

Reply:
I would like to see proof of the use of "wrapper" in technical literature. =)

Reply:
> I would like to see proof of the use of "обертка" in technical literature. =)

Articles (only about `indexedDb` as example):

* https://habr.com/ru/post/138987/
* https://habr.com/ru/post/213515/
* https://html5.by/blog/localforage/

Documentation (about PostgreSQL):

* https://postgrespro.ru/docs/postgrespro/11/fdwhandler

Reply:
ok, now i see. I agree.
note that in 3 out of 4 cases the word "обЕртка" is used =)

fixed

Reply:
> note that in 3 out of 4 cases the word "обЕртка" is used =)

Oh, these lazy people… I'm working on this. :wink:

Reply:
@Jollfye @AlexWayfer 
+for?
![image](https://user-images.githubusercontent.com/23438485/52485115-2030ca00-2bc9-11e9-9a2f-7a5dc40d731a.png)

Reply:
I think db.js is "обёртка indexedDb" (indexedDb wrapper) so no "для" (for) is needed here but let's hear what @AlexWayfer may say 🤔

Reply:
From [the official repo](https://github.com/aaronpowell/db.js/):

> db.js is a wrapper for IndexedDB

So, literally it translates with "для". Yeah, Russian is very powerful and flexible language.

And technically the current text can be correct. 

But the more clear (for me) would be:

> Также используется обёртка для indexedDb — db.js, от Aaron Powell.

Or:

> Также используется db.js — обёртка для indexedDb от Aaron Powell.

Reply:
I vote for the first variant, so "html_about_indexedDb_prefix" should be:
"Также используется обёртка для indexedDb — "
and "html_about_indexedDb_suffix":
", от Aaron Powell."
but there's may be a problem, coz live it may look like:
Также используется обёртка для indexedDb — db.js , от Aaron Powell.
extra space and this space is not in "html_about_indexedDb_suffix".
Also I'm not sure bout em dash —, maybe there should be en dash –?

Reply:
I, as always, are against symbols that cannot be typed using the keyboard...

Reply:
> Also I'm not sure bout em dash —, maybe there should be en dash –?

I'm quite sure that there should be em dash: https://en.wikipedia.org/wiki/Dash

Also:

> В правилах русской пунктуации и типографики упоминается единый знак «тире». В терминах компьютерных технологий (пришедших из английской типографики) он соответствует так называемому длинному тире (англ. em dash).

[Source](https://ru.wikipedia.org/wiki/%D0%A2%D0%B8%D1%80%D0%B5)

---

> I, as always, are against symbols that cannot be typed using the keyboard...

I typed it via keyboard, just use [Compose Key](https://en.wikipedia.org/wiki/Compose_key). :wink:

Reply:
Alright then, on @MonsterNya's discretion.